src/modules/ens/ens.service.ts

```diff
@@ -1,26 +1,30 @@
 import { Injectable } from '@nestjs/common';
 import { SchedulerRegistry } from '@nestjs/schedule';
 import { utils, errors } from 'ethers';
-import { abi as ensResolverContract } from '@ensdomains/resolver/build/contracts/PublicResolver.json';
 import { PublicResolverFactory } from '../../ethers/PublicResolverFactory';
 import { RoleService } from '../role/role.service';
 import { ApplicationService } from '../application/application.service';
 import { OrganizationService } from '../organization/organization.service';
 import { EnsRegistryFactory } from '../../ethers/EnsRegistryFactory';
 import { ConfigService } from '@nestjs/config';
 import { PublicResolver } from '../../ethers/PublicResolver';
+import { DomainNotifier } from '../../ethers/DomainNotifier';
+import { DomainNotifierFactory } from '../../ethers/DomainNotifierFactory';
```

Comment:
Hmm, I'm not the biggest fan of that because I think the contracts types are generated with specific parameters (such as typescript and ethers version) so it would be hard for `iam-contracts` to provide all of the possible types. Right?